topics/imaging/tutorials/voronoi-segmentation/tutorial.md

```diff
@@ -24,15 +24,20 @@ requirements:
     topic_name: imaging
     tutorials:
       - imaging-introduction
-contributors:
-- evenmm
-- rmassei
-- kostrykin
-- annefou
+
+contributions:
+  authorship:
+    - evenmm
+    - rmassei
+    - kostrykin
+    - annefou
+  funding:
+    - OSCARS-FIESTA
```

Comment:
this ID should match the key in the ORGANISATIONS.yaml file, so either change this to `fiesta`, or change the ID in the organisations file to match this

```suggestion
    - fiesta
```